Tools/LambdaTestTool/src/Amazon.Lambda.TestTool.BlazorTester/Amazon.Lambda.TestTool.BlazorTester.csproj

```diff
@@ -26,7 +26,11 @@
 	
     <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0' ">
 		<PackageReference Include="Microsoft.Extensions.FileProviders.Embedded" Version="6.0.0" />
-    </ItemGroup>	
+    </ItemGroup>
+
+    <ItemGroup Condition=" '$(TargetFramework)' == 'net7.0' ">
+		<PackageReference Include="Microsoft.Extensions.FileProviders.Embedded" Version="7.0.0-rc.2.22476.2" />
```

Comment:
Do we need to specify an exact version here? or can we use "7.0.0" to avoid having to revisit this when .NET 7 GAs?

Reply:
Because there is no 7 version without a suffix like `-rc2` wildcarding the version doesn't work. We will just have to do a future release switching to the GA version.